apiservers/engine/backends/network.go

```diff
@@ -114,7 +114,7 @@ func (n *Network) CreateNetwork(name, driver string, ipam apinet.IPAM, options m
 		IPAM:      pools,
 	}
 
-	created, err := PortLayerClient().Scopes.Create(scopes.NewCreateParams().WithConfig(cfg))
+	created, err := PortLayerClient().Scopes.CreateScope(scopes.NewCreateScopeParams().WithConfig(cfg))
```

Comment:
Minor naming quirk - we've just altered the ContainerCreate call to be containers.Create - why are we going the other way with CreateScope?